Packs/CommonPlaybooks/Playbooks/playbook-Block_IP_-_Generic_v3.yml

```diff
@@ -0,0 +1,3569 @@
+id: Block IP - Generic v3
+version: -1
+name: Block IP - Generic v3
+description: "This playbook blocks malicious IP addresses using all integrations that\
```

Comment:
Notice how to describe the PAN-OS various options and that the user may not need to enable all of them